cirq/google/optimizers/convert_to_sycamore_gates.py

```diff
@@ -201,6 +212,87 @@ def known_two_q_operations_to_sycamore_operations(
         raise ValueError("Unrecognized gate: {!r}".format(op))
 
 
+def decompose_phased_iswap_into_syc(phase_exponent: float, a: ops.Qid,
+                                    b: ops.Qid):
+    """Decompose PhasedISwap with an exponent of 1.
+
+    This should only be called if the Gate has an exponent of 1 - otherwise,
+    decompose_phased_iswap_into_syc_precomputed should be used instead. The
+    advantage of using this function is that the resulting circuit will be
+    smaller.
+
+    Args:
+        phase_exponent: The exponent on the Z gates.
+        a: First qubit id to operate on
+        b: Second qubit id to operate on
+    Returns:
+        a Cirq program implementing the Phased ISWAP gate
+
+    """
+
+    yield ops.Z(a)**phase_exponent,
+    yield ops.Z(b)**-phase_exponent,
+    yield decompose_iswap_into_syc(a, b),
+    yield ops.Z(a)**-phase_exponent,
+    yield ops.Z(b)**phase_exponent,
+
+
+def decompose_phased_iswap_into_syc_precomputed(theta: float, a: ops.Qid,
```

Comment:
Is this function covered by a test?

Reply:
I think the givens rotation does, but I could be wrong. Added an explicit test for it.